cypress/support/cypress-commands/assertion.js

```diff
@@ -639,7 +652,7 @@ Cypress.Commands.add(
 
     fireLog.info('Event Received Check : ' + eventReceivedCheck);
     fireLog.info('Event Schema Check : ' + schemaCheck);
-    fireLog.info('Event Content Check : ' + contentCheck);
+    // fireLog.info('Event Content Check : ' + contentCheck);
```

Comment:
If we don't need this code can we remove it?

Reply:
removed